.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,15 @@
+---
+name: issue_template
+about: The standard template to use
+title: ""
+labels: ''
+assignees: ''
+
+---
+
+A brief description of the issue.
+
+**Does this issue describe a bug that could be breaking child projects right now? If so please make other people aware via the #python slack channel ASAP**
```

Comment:
Possibly it should be "... could be breaking all skeleton projects ...", given this will likely be seen when making an issue on a specific repo?

Reply:
agreed